src/public/place/directive.html

```diff
@@ -0,0 +1,29 @@
+<div class="columns" id="directive" hx-swap-oob="true">
+  <div class="column is-10">
+    <section class="hero box is-small is-light">
+      <div class="hero-body">
+        {% if progress.inProgressCount > 0 %}
+          {% include "place/directive_3_in_progress.html" %}
+        {% elsif progress.successCount > 0 %}
```

Comment:
We should show the upload prompt anytime we have anything staged

Reply:
In the nav bar or in this "directive" page?  If we are showing it all the time, I'm inclined to use the navbar.

Reply:
In the directive, if we upload successfully we get the download prompt. If we stage new changes after the initial upload, I'd expect the download prompt to change to the upload prompt again

Reply:
ok. so you don't want to show both at once.

what if you upload 300 and there are 3 failures? should they stilll see upload, download, or something else?

Reply:
I think download with an option to retry failed works in that case